docs/knowledgebase/advanced/account-info.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Account Information
+---
+
+## `Account` StorageMap
+
+In Substrate, account information is stored in a
+[**Storage Map**](https://substrate.dev/rustdocs/v3.0.0/frame_support/storage/types/struct.StorageMap.html#impl),
+extracted and shown below:
+
+[`frame/system/src/lib.rs`](https://github.com/paritytech/substrate/blob/bcd649ffca9efc93f8b4ac1506ec8117b71e1aac/frame/system/src/lib.rs#L530-L538):
```

Comment:
linking to a specific commit and lines of code probably not the best. I think just point to the file from Master

Reply:
I am now referring to the API doc page for src code line number, and they are less likely to be outdated.